src/components/pages/Registration/HowManyStudents.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import '../../../styles/registration.less';
+import RegistrationProgress from './RegistrationProgress';
+
+export default function HowManyStudents() {
+  const [numStudents, setNumStudents] = useState(1);
+
+  const handleChange = e => {
+    setNumStudents(e.target.value);
+  };
+
+  return (
+    <div className="reg-content-container email-confirmation">
+      <RegistrationProgress step_num={1} />
+
+      <div className="content">
+        <p className="color-one">
+          How many student profiles would you like to create?
+        </p>
+        <form>
+          <input
+            className="student-num"
+            type="number"
+            value={numStudents}
+            onChange={handleChange}
+          />
+        </form>
+
+        <div className="link-container">
+          <a href="#">Create Profiles</a>
+          {/* update href after finding correct path from Okta */}
```

Comment:
Just to clarify, we only use Okta for the login and registration forms. This will link to the next page /registration-3 which is not yet built so the blank href is fine for now but keep in mind where this needs to link to for the future.

Reply:
I see. As for passing the value. maybe it would be easier to add it to local storage and then removed on the next page. This will make sure that it is still accessible even if they close they're browser and come back later.